Funky-Chickens/Bangazon-CLI
# Cm payments ctrl

What was changed:
- Payment types added to user profile
- make sure order is open to add payment type
- calculate and display total
- user selects payment for order
- payment type is saved to DB

TO TEST:

1. `npm run db:reset`
2. `npm start`
3. Select a customer (maybe 1),
4. select option 3 (complete an order). 
5. Choose a payment type. 
6. Check DB browser to see if the NULL has changed to a payment id.

#6
